GCTC-NTGC/gc-digital-talent
# Bug - Fixes `SkillBlock` definition click handler

Resolves #4577.

## Locations
This component can be found the following locations to be tested:

- `/search`
- `/users/{userId}/profile/experiences/{experienceType}/create`
- `/users/{userId}/profile/experiences/{experienceType}/{experienceId}/edit`
- `/admin/pools/{poolId}/edit`

## Steps to test

1.  Build admin `npm run production --workspace=admin`
2. Build talentsearch `npm run production --workspace=talentsearch`
3.  Navigate to one of the locations listed in the "Locations" section
4.  Select "See definition" and confirm the definition appears
5.  Confirm you can hide the definition and the text reads "Hide definition"
